debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/SqlUtilsTest.java

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
@Naros Should not be there a replacement for delete test segments? Maybe the flush strategies needs their own tests?

Reply:
I'll take a look at and see if it's possible but a number of these were removed simply due to dead-code and no other reason.